### DISCUSSION TITLE
Max Number of Briefcases

### DISCUSSION THREAD

**Discussion by nn15296** [iTwin/itwinjs-core#1298] [Q&A] [answered]
Hello

I am getting the following error when trying to download a briefcase:

`Error   |imodelhub-client.iModelHub| 400 iModelHub.MaximumNumberOfBriefcasesPerUser: Maximum number of briefcases per user reached.`

What steps can be taken to "release"/reuse some of these briefcases because I've deleted them locally.

Thanks

**Comment by ramanujam-raman-zz** [comment, marked as answer]
What is the API that you are using that led you to this? These represent abandoned briefcases. 

You can release them to get past the error, but it would be good to trace why it happened. Use the method [here ](https://github.com/imodeljs/imodeljs/blob/master/core/backend/src/test/integration/HubUtility.ts)- purgeAcquiredBriefcases. 

Note that releasing briefcases would cause the iModelHub to stop tracking the briefcase, and you cannot push changes from it.

**Reply by ColinKerr** [reply]
@nn15296 if you are using the [BriefcaseManager.downloadBriefcase](https://www.itwinjs.org/reference/imodeljs-backend/imodels/briefcasemanager/downloadbriefcasestatic/#downloadbriefcase) method you can set the briefcaseId to 0 or [BriefcaseIdValue.Standalone](https://www.itwinjs.org/reference/imodeljs-common/imodels/briefcaseidvalue/).  This will keep you from reserving a briefcase id and is the best option if you do not want to edit the iModel.  If you want to edit you must reserve a briefcase id and you must manage the id's lifetime.

```typescript
const requestNewBriefcaseArg: RequestNewBriefcaseProps = { contextId: contextId, iModelId: imodelId, asOf: version.toJSON(), briefcaseId: 0 };
const briefcaseProps = await BriefcaseManager.downloadBriefcase(context, requestNewBriefcaseArg);
```
